test/e2e/pages/page.js

```diff
@@ -70,6 +70,19 @@ export default class Page {
       .notContains('/auth/login')
   }
 
+  /**
+   * Select "all locations"
+   *
+   * @returns {Promise}
+   */
+  async chooseAllLocations() {
+    const allLocationsLink = Selector('main p a').withAttribute(
```

Comment:
Maybe just use `Selector('a')` here. It means it's less likely to break if we move the position of that element in the DOM.